### ISSUE TITLE
Waiting for App Store Connect to finish processing the new build

### ISSUE THREAD

**Issue by danimal521** [fastlane/fastlane#16696]
Hey Guys,

Anyone getting a timeout on "Waiting for App Store Connect to finish processing the new build "

This used to work great before all those recent app store connect changes. 

I am using azure devops and the appstore release task, here is my yaml:

    - task: AppStoreRelease@1
      inputs:
        serviceEndpoint: 'Apple'
        appIdentifier: 'com.xxx'
        fastlaneToolsVersion: SpecificVersion
        fastlaneToolsSpecificVersion: 2.150.0.rc3

2020-06-16T23:41:19.1547190Z ##[section]Starting: AppStoreRelease
2020-06-16T23:41:19.1561360Z ==============================================================================
2020-06-16T23:41:19.1561780Z Task         : Apple App Store Release
2020-06-16T23:41:19.1562150Z Description  : Release an app to TestFlight or the Apple App Store
2020-06-16T23:41:19.1562470Z Version      : 1.158.0
2020-06-16T23:41:19.1562750Z Author       : Microsoft Corporation
2020-06-16T23:41:19.1563210Z Help         : [More Information](https://marketplace.visualstudio.com/items?itemName=ms-vsclient.app-store)
2020-06-16T23:41:19.1563710Z ==============================================================================
2020-06-16T23:41:19.7910840Z 6bee70df-d608-4eda-aa8b-5033bfa2c84c exists true
2020-06-16T23:41:20.5983300Z [command]/usr/local/opt/ruby/bin/gem install fastlane
2020-06-16T23:41:20.6058650Z (node:2263) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-06-16T23:41:20.6060920Z (node:2263) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-06-16T23:41:20.6062940Z (node:2263) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-06-16T23:41:20.6064070Z (node:2263) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-06-16T23:41:20.6065240Z (node:2263) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-06-16T23:41:20.6066510Z (node:2263) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-06-16T23:41:20.6067610Z (node:2263) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-06-16T23:41:20.6068760Z (node:2263) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-06-16T23:41:20.6069890Z (node:2263) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-06-16T23:41:20.6071530Z (node:2263) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-06-16T23:41:20.6072680Z (node:2263) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-06-16T23:41:20.6073760Z (node:2263) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-06-16T23:41:20.6074900Z (node:2263) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-06-16T23:41:20.6076060Z (node:2263) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-06-16T23:41:20.6077130Z (node:2263) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-06-16T23:41:20.6078310Z (node:2263) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-06-16T23:41:20.6079390Z (node:2263) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-06-16T23:41:20.6080540Z (node:2263) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-06-16T23:41:20.6081690Z (node:2263) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-06-16T23:41:20.6082780Z (node:2263) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-06-16T23:41:20.6083920Z (node:2263) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-06-16T23:41:20.6085120Z (node:2263) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-06-16T23:41:20.6086210Z (node:2263) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-06-16T23:41:20.6087340Z (node:2263) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-06-16T23:41:20.6088660Z (node:2263) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-06-16T23:41:20.6089830Z (node:2263) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-06-16T23:41:20.6090920Z (node:2263) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-06-16T23:41:20.6091730Z (node:2263) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-06-16T23:41:20.6092550Z (node:2263) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-06-16T23:41:51.5964340Z Successfully installed fastlane-2.149.1
2020-06-16T23:41:51.5965340Z Parsing documentation for fastlane-2.149.1
2020-06-16T23:41:51.5966070Z Installing ri documentation for fastlane-2.149.1
2020-06-16T23:41:51.5966430Z Done installing documentation for fastlane after 22 seconds
2020-06-16T23:41:51.5966750Z 1 gem installed
2020-06-16T23:41:51.6259760Z [command]/usr/local/opt/ruby/bin/gem update fastlane -i /Users/runner/.gem-cache
2020-06-16T23:41:54.4682390Z Updating installed gems
2020-06-16T23:41:54.4683700Z Nothing to update
2020-06-16T23:41:54.4966960Z [command]fastlane pilot upload -u *** -i /Users/runner/runners/2.170.1/work/1/s/xxx.iOS/bin/iPhone/Release/xxx.iOS.ipa -a com.xxx
2020-06-16T23:41:59.7390880Z [23:41:59]: [33mSeems like launching fastlane takes a while - please run[0m
2020-06-16T23:41:59.7449400Z [23:41:59]: 
2020-06-16T23:41:59.7450410Z [23:41:59]: [36m$ [sudo] gem cleanup[0m
2020-06-16T23:41:59.7450750Z [23:41:59]: 
2020-06-16T23:41:59.7451480Z [23:41:59]: [33mto uninstall outdated gems and make fastlane launch faster[0m
2020-06-16T23:41:59.7452450Z [23:41:59]: [33mAlternatively it's recommended to start using a Gemfile to lock your dependencies[0m
2020-06-16T23:41:59.7453240Z [23:41:59]: [33mTo get started with a Gemfile, run[0m
2020-06-16T23:41:59.7453590Z [23:41:59]: 
2020-06-16T23:41:59.7454200Z [23:41:59]: [36m$ bundle init[0m
2020-06-16T23:41:59.7454880Z [23:41:59]: [36m$ echo 'gem "fastlane"' >> Gemfile[0m
2020-06-16T23:41:59.7455600Z [23:41:59]: [36m$ bundle install[0m
2020-06-16T23:41:59.7455860Z [23:41:59]: 
2020-06-16T23:41:59.7456650Z [23:41:59]: [33mAfter creating the Gemfile and Gemfile.lock, commit those files into version control[0m
2020-06-16T23:41:59.7457670Z [23:41:59]: [33mGet started using a Gemfile for fastlane https://docs.fastlane.tools/getting-started/ios/setup/#use-a-gemfile[0m
2020-06-16T23:42:07.0165060Z [23:42:07]: Sending anonymous analytics information
2020-06-16T23:42:07.0205510Z [23:42:07]: Learn more at https://docs.fastlane.tools/#metrics
2020-06-16T23:42:07.0206130Z [23:42:07]: No personal or sensitive data is sent.
2020-06-16T23:42:07.0207320Z [23:42:07]: You can disable this by adding `opt_out_usage` at the top of your Fastfile
2020-06-16T23:42:07.0362190Z [23:42:07]: Login to App Store Connect (***)
2020-06-16T23:42:09.4484090Z [23:42:09]: Login successful
2020-06-16T23:42:10.8529670Z [23:42:10]: Ready to upload new build to TestFlight (App: 1493006778)...
2020-06-16T23:42:11.0051580Z [23:42:11]: Going to upload updated app to App Store Connect
2020-06-16T23:42:11.0056160Z [23:42:11]: This might take a few minutes. Please don't interrupt the script.
2020-06-16T23:44:37.7378540Z [23:44:37]: iTunes Transporter successfully finished its job
2020-06-16T23:44:37.9622900Z [23:44:37]: --------------------------------------------------------------------
2020-06-16T23:44:37.9624120Z [23:44:37]: Successfully uploaded package to App Store Connect. It might take a few minutes until it's visible online.
2020-06-16T23:44:37.9625070Z [23:44:37]: --------------------------------------------------------------------
2020-06-16T23:44:37.9633740Z [23:44:37]: Successfully uploaded the new binary to App Store Connect
2020-06-16T23:44:37.9635090Z [23:44:37]: If you want to skip waiting for the processing to be finished, use the `skip_waiting_for_build_processing` option
2020-06-16T23:44:37.9636240Z [23:44:37]: Note that if `skip_waiting_for_build_processing` is used but a `changelog` is supplied, this process will wait for the build to appear on AppStoreConnect, update the changelog and then skip the remaining of the processing steps.
2020-06-16T23:44:38.0080010Z [23:44:38]: Waiting for processing on... app_id: 1493006778, app_version: 1.7.8, build_version: 1.7.8, platform: IOS
2020-06-16T23:44:38.4592840Z [23:44:38]: Read more information on why this build isn't showing up yet - https://github.com/fastlane/fastlane/issues/14997
2020-06-16T23:44:38.4595720Z [23:44:38]: Waiting for the build to show up in the build list - this may take a few minutes (check your email for processing issues if this continues)
2020-06-16T23:45:08.8767160Z [23:45:08]: Waiting for the build to show up in the build list - this may take a few minutes (check your email for processing issues if this continues)
2020-06-16T23:45:39.0946930Z [23:45:39]: Waiting for the build to show up in the build list - this may take a few minutes (check your email for processing issues if this continues)
2020-06-16T23:46:09.3419270Z [23:46:09]: Waiting for the build to show up in the build list - this may take a few minutes (check your email for processing issues if this continues)
2020-06-16T23:46:42.5475880Z [23:46:42]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-16T23:47:13.2330860Z [23:47:13]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-16T23:47:43.9034150Z [23:47:43]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-16T23:48:14.3262160Z [23:48:14]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-16T23:48:44.8890670Z [23:48:44]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-16T23:49:15.4052870Z [23:49:15]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-16T23:49:45.6820970Z [23:49:45]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-16T23:50:15.9701160Z [23:50:15]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-16T23:50:46.4433920Z [23:50:46]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-16T23:51:17.4251370Z [23:51:17]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-16T23:51:47.6886050Z [23:51:47]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-16T23:52:18.2137540Z [23:52:18]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-16T23:52:48.8580220Z [23:52:48]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-16T23:53:19.4279750Z [23:53:19]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-16T23:53:49.7063580Z [23:53:49]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-16T23:54:20.2390440Z [23:54:20]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-16T23:54:50.9856400Z [23:54:50]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-16T23:55:21.3769900Z [23:55:21]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-16T23:55:51.7139470Z [23:55:51]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-16T23:56:22.0942410Z [23:56:22]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-16T23:56:52.7228860Z [23:56:52]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-16T23:57:23.3245360Z [23:57:23]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-16T23:57:53.7249350Z [23:57:53]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-16T23:58:24.0484740Z [23:58:24]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-16T23:58:54.6290160Z [23:58:54]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-16T23:59:24.9952800Z [23:59:24]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-16T23:59:55.3287070Z [23:59:55]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:00:25.9140650Z [00:00:25]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:00:56.5008870Z [00:00:56]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:01:26.9032320Z [00:01:26]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:01:57.3009590Z [00:01:57]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:02:27.6488950Z [00:02:27]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:02:58.4782700Z [00:02:58]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:03:28.7487750Z [00:03:28]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:03:59.0107120Z [00:03:59]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:04:29.2848400Z [00:04:29]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:04:59.6340080Z [00:04:59]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:05:30.0395060Z [00:05:30]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:06:00.8183020Z [00:06:00]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:06:31.2328070Z [00:06:31]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:07:01.5549810Z [00:07:01]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:07:31.9642970Z [00:07:31]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:08:02.2515930Z [00:08:02]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:08:32.8977300Z [00:08:32]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:09:03.4845100Z [00:09:03]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:09:33.8744680Z [00:09:33]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:10:04.3883420Z [00:10:04]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:10:34.9086050Z [00:10:34]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:11:05.2755090Z [00:11:05]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:11:35.8598400Z [00:11:35]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:12:06.5124090Z [00:12:06]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:12:36.8089770Z [00:12:36]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:13:07.1031850Z [00:13:07]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:13:37.4137800Z [00:13:37]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:14:08.0041260Z [00:14:08]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:14:38.2855350Z [00:14:38]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:15:08.6122010Z [00:15:08]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:15:39.0163970Z [00:15:39]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:16:09.6770870Z [00:16:09]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:16:40.0944960Z [00:16:40]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:17:10.3774470Z [00:17:10]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:17:40.6507500Z [00:17:40]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:18:10.9617180Z [00:18:10]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:18:41.3095820Z [00:18:41]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:19:11.6692750Z [00:19:11]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:19:42.2370690Z [00:19:42]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:20:12.5955100Z [00:20:12]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:20:42.9257190Z [00:20:42]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:21:13.5639650Z [00:21:13]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:21:44.0793730Z [00:21:44]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:22:14.3925330Z [00:22:14]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:22:45.0125230Z [00:22:45]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:23:15.4526720Z [00:23:15]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:23:46.0732570Z [00:23:46]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:24:16.4047540Z [00:24:16]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:24:46.7561170Z [00:24:46]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:25:17.1837380Z [00:25:17]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:25:47.4666160Z [00:25:47]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:26:18.1768960Z [00:26:18]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:26:48.5573150Z [00:26:48]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:27:19.1148100Z [00:27:19]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:27:49.7712380Z [00:27:49]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:28:20.3862500Z [00:28:20]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:28:50.9211460Z [00:28:50]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:29:21.4545580Z [00:29:21]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:29:52.0948840Z [00:29:52]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:30:22.3489930Z [00:30:22]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:30:52.6977040Z [00:30:52]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:31:23.3019680Z [00:31:23]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:31:53.8089390Z [00:31:53]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:32:24.1002920Z [00:32:24]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:32:54.4091910Z [00:32:54]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:33:24.7462940Z [00:33:24]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:33:55.3420340Z [00:33:55]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:34:26.4994290Z [00:34:26]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:34:56.8757140Z [00:34:56]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:35:27.6135240Z [00:35:27]: Waiting for App Store Connect to finish processing the new build (1.7.8 - 1.7.8) for IOS
2020-06-17T00:35:43.2918920Z ##[error]The operation was canceled.
2020-06-17T00:35:43.2933140Z ##[section]Finishing: AppStoreRelease

**Comment by joshdholtz** [comment]
@danimal521 👋 Hey! Processing can sometimes take a really long time 🤔 Did you go to the App Store Connect dashboard to verify that it still showed as processing there?

**Comment by janpio** [comment]
With 2.150.0 released you should also upgrade and see if the problem persists.

**Comment by BytesGuy** [comment]
Along similar lines, I am hitting this when uploading with Pilot. It actually gets pinged for finishing the build processing,  but never returns:

```
[11:46:54]: --- Step: pilot ---
[11:46:54]: -------------------
[11:46:54]: Login to App Store Connect ([redacted])
[11:46:55]: Login successful
[11:46:57]: Ready to upload new build to TestFlight (App: [redacted])...
[11:46:57]: Going to upload updated app to App Store Connect
[11:46:57]: This might take a few minutes. Please don't interrupt the script.
[11:48:28]: iTunes Transporter successfully finished its job
[11:48:28]: --------------------------------------------------------------------
[11:48:28]: Successfully uploaded package to App Store Connect. It might take a few minutes until it's visible online.
[11:48:28]: --------------------------------------------------------------------
[11:48:28]: Successfully uploaded the new binary to App Store Connect
[11:48:28]: If you want to skip waiting for the processing to be finished, use the `skip_waiting_for_build_processing` option
[11:48:28]: Note that if `skip_waiting_for_build_processing` is used but a `changelog` is supplied, this process will wait for the build to appear on AppStoreConnect, update the changelog and then skip the remaining of the processing steps.
[11:48:28]: Waiting for processing on... app_id: [redacted], app_version: 1.2, build_version: 5, platform: IOS
[11:48:28]: Read more information on why this build isn't showing up yet - https://github.com/fastlane/fastlane/issues/14997
[11:48:28]: Waiting for the build to show up in the build list - this may take a few minutes (check your email for processing issues if this continues)
[11:48:59]: Waiting for the build to show up in the build list - this may take a few minutes (check your email for processing issues if this continues)
[11:49:32]: Waiting for App Store Connect to finish processing the new build (1.2 - 5) for IOS
[11:50:03]: Successfully finished processing the build 1.2 - 5 for IOS
 ```

It will then just sit there until Fastlane is manually killed.

If this warrants a fresh issue please let me know.

**Comment by KennethBass** [comment]
Processing can sometimes take a really long time 🤔 Did you go to the App Store Connect dashboard to verify that it still showed as processing there?

**Comment by SusanWong15** [comment]
With 2.150.0 released you should also upgrade and see if the problem persists.

**Comment by BytesGuy** [comment]
I managed to resolve my issue. It was due to some new agreements being made available, so I had to sign in to the Apple ID I use for CircleCI and log in to the dev portal and agree. I'm not sure if it's possible for Fastlane to detect any outstanding agreements, but it would certainly be beneficial to have an error message that suggests this. Hopefully this helps the OP too

**Comment by reishka** [comment]
FWIW, We're not using CircleCI, and over the past week every build I've attempted to upload has stalled on "waiting to finish processing". The build finishes processing in the portal in under 5 minutes, yet fastlane will churn indefinitely until I cancel it (we've waited an hour, just to see what would happen, and no dice). We're on  2.156.1

**Comment by reishka** [comment]
I discovered why this continually spins in our environment. 

Our public version number is often times different from the internal version number. Previously, we could supply a version number, and automatically select the most recently uploaded build (whatever version that happened to be). 

Now, when we supply a version number (this gets set as the public version number in the edit version), it looks for a corresponding build with the same version number. 

ie: We supply version "10.0.0", and it looks for build "10.0.0.x", where x is the most recent build. This may or may not match the version number that we're actually releasing. If it doesn't, it'll spin forever. 

We worked around this by not setting a version number at all, and then calling deliver a second time and supplying ONLY the version number (no other metadata, and performing no other operations). This lets us select the most recently uploaded build no matter the version/build number, and it lets us set the public-facing version number to whatever we need it to be.
